services/environments/minigrid_test_fullEnv.py

```diff
@@ -0,0 +1,77 @@
+from gym_minigrid.minigrid import Floor
+from gym_minigrid.envs.empty import EmptyEnv
+from gym_minigrid.register import register
+from gym import spaces
+
+import statistics, sys
+import numpy
+from enum import IntEnum
+from services.constants import *
+
+class Arie5x5(EmptyEnv):
+    class Actions(IntEnum):
+          # Turn left, turn right, move forward
+          left = 0
+          right = 1
+          forward = 2
+
+    def __init__(self, **kwargs):
+        super().__init__(size=5, **kwargs)
+        # Action enumeration for this environment
+        self.actions = Arie5x5.Actions
+
+        # Actions are discrete integer values
+        self.action_space = spaces.Discrete(len(self.actions))
+
+        self.metadata[CONSTRAINT_VIOLATION_COUNT] = 0
+        self.metadata[EPISODE_ACTION_HISTORY] = []
+
+    def remember_size(self, width, height):
```

Comment:
This is unnecessary.  We already have access to this info in self.width and self.height